tox.ini

```diff
@@ -1,10 +1,21 @@
 [tox]
 envlist =
-    py26,py27
+    py27,
+    py35,
+    py36,
 
 [testenv]
 deps =
     zope.testrunner
```

Comment:
I like to list zope.testrunner in the `[test]` extra. It simplifies both tox.ini and .travis.yml, and (to me at least) it helps make it more clear that we're unlikely to run with 'python setup.py test'.